app/assets/stylesheets/components/form/_form.scss

```diff
@@ -5,3 +5,18 @@
 .form__input-container--medium {
   max-width: 140px;
 }
+
+%form-row-2up {
```

Comment:
Did you see the on-focus tooltips that appear on the mortgage calculator? Its for reasons like that that we left the space to the side of a form field spare. 

People already don't know where to look for canonical examples of Dough components, and even though this is an admin interface, it's still MAS code. Creating another deviation (particularly around forms, arguably the most complex bit of Dough) will just add more confusion. I'd really like to try and limit it for maintainability's sake if we can. 

Happy to be pragmatic about it, but do want to raise a concern around this design.